Fix:

long-options.c:65:4: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

error.c:172:36: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]

lsyslog.c:67:4: error: implicitly declaring library function 'sprintf' with type 'int (char *, const char *, ...)' [-Werror,-Wimplicit-function-declaration]

--- lib/long-options.c.orig	1998-04-26 15:22:35.000000000 +0200
+++ lib/long-options.c	2020-11-07 05:45:08.000000000 +0100
@@ -21,6 +21,7 @@
 #include <config.h>
 #endif
 
+#include <stdlib.h>
 #include <stdio.h>
 #include <getopt.h>
 #include "long-options.h"
--- lib/error.c.orig	2020-11-07 05:46:00.000000000 +0100
+++ lib/error.c	2020-11-07 05:46:21.000000000 +0100
@@ -21,6 +21,7 @@
 # include <config.h>
 #endif
 
+#include <string.h>
 #include <stdio.h>
 
 #if HAVE_VPRINTF || HAVE_DOPRNT || _LIBC

--- src/lsyslog.c.orig	1998-12-29 18:27:46.000000000 +0100
+++ src/lsyslog.c	2020-11-07 06:00:35.000000000 +0100
@@ -22,6 +22,7 @@
 #ifdef ENABLE_SYSLOG
 #include "zglobal.h"
 #include <pwd.h>
+#include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #endif
